src/amo/reducers/home.js

```diff
@@ -126,12 +130,13 @@ const reducer = (
       return {
         ...state,
         collections: collections.map((collection) => {
-          if (collection) {
-            return collection.results
-              .slice(0, LANDING_PAGE_ADDON_COUNT)
-              .map((item) => {
-                return createInternalAddon(item.addon);
-              });
+          if (collection && collection.results && collection.results.length) {
```

Comment:
We need a test for the case where a collection exists, but it is empty, in which case we expect a `null` to be returned.

Reply:
just checking...this won't suffice? https://github.com/mozilla/addons-frontend/blob/master/tests/unit/amo/reducers/test_home.js#L60

Reply:
No, in that case `collection` is `null`, but we need a test for where a collection exists, but is empty.